the-graph/the-graph-app.js

```diff
@@ -159,8 +159,8 @@ module.exports.register = function (context) {
         scale: fit.scale,
         width: this.props.width,
         height: this.props.height,
-        minZoom: this.props.minZoom,
-        maxZoom: this.props.maxZoom,
+        minZoom: this.props.minZoom || 0.15,
```

Comment:
Or alternatively that an invalid value for props.minZoom is getting in from some other source.